### ISSUE TITLE
Add auto update for Filebot & Rename Reinstall

### ISSUE THREAD

**Issue by Marbellasaad** [bytesizedhosting/bcd#39]
Hello,on bytesized connect when you hit reinstall it will not get the last update of filebot but always the 4.7.2. So you can't update filebot and always have to wait the image to be updated.

Actually you can update filebot manually by spawning into the docker and doing these commands :
wget -O /app/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.8'
dpkg -i /app/filebot.deb && rm /app/filebot.deb

It seem likes filebot are the only app who doesn't get the last update (in dockerfile) when you install the docker.

Exemples : Sickrage get the last update available on github :
https://puu.sh/uyZko/d2d35d5cd1.png

And it's the same for almost every bsc app excluding filebot.

To easily fix that,just modify "https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.2" in dockerfile by ""https://www.filebot.net/download.php?mode=s&type=deb&arch=amd64"

(BTW if you could rename "Reinstall" by update in bytesized connect if would be more intuitive).

And a last request,can you tell me where did you found your "monitor"/"watcher" plugin for filebot ?
Thanks !

**Comment by maran** [comment]
I didn't consider auto-upgrading for Filebot since I don't think the version matters that lot for Filebot. I've been using 4.7.2 for over two years now without any problems. 

The problem with the option you suggested is that it just hardcodes the version to the current version, one that will expire again at one point. 

I'm open for adding auto-update but it should always update to the latest available version if we go that route. If you have a way to achieve that I'm open to implementing it.

**Comment by Marbellasaad** [comment]
Yeah,but then when we reinstall,we would got the latest version,isn't reinstall the way to update app in bytesized connect? Because app like plex and rutorrent need a reinstall to be updated. (or plex/rutorrent have an auto update feature,but i haven't seen anything like it,it would be cool if all apps on this planet would be like sickrage and have an auto update via GUI feature)
